Plugin~/WebRTCPlugin/GraphicsDevice/Vulkan/VulkanUtility.cpp

```diff
@@ -0,0 +1,377 @@
+﻿#include "pch.h"
```

Comment:
I think this code less error check.
We need to check the return values from Vulkan API.

Reply:
Which Vulkan API are you implying here ?